.github/workflows/bats.yml

```diff
@@ -32,6 +28,7 @@ jobs:
         HEAD_REF: ${{ github.head_ref }}
       run: |
         # when running in test-harness, need to mark the directory safe for git operations
+        echo "/usr/share/terraform/1/bin/terraform" >> "$GITHUB_PATH"
```

Comment:
This should not be necessary, because we should continue to be (and currently are) installing a link to the default version, even if it is tofu, as `/usr/bin/terraform`. 

```suggestion
```

Reply:
Unfortunately, we can not do as usual.  Terratest fallback to opentofu if terraform binary not found. 
Otherwise, we must update all terraform modules `go` code to support explicit binary path.

Reply:
This workflow is not running Terratest, it is running `bats`, which uses whatever `terraform` is installed.